### PR TITLE
[DOC canary] Documentation added for ember testing helper scripts

### PR DESCRIPTION
**Pull request by witty123**
Added documentation for and_then.js and current_url.js in ember testing helper scripts. Per issue 
#13862

### CODE REVIEW

File: packages/ember-testing/lib/helpers/current_url.js [hunk 1]
```diff
@@ -1,3 +1,18 @@
+/**
```
Comment (rwjblue) [inline packages/ember-testing/lib/helpers/current_url.js]: Can you move this comment down to just above the `currentURL` function export below?

**Comment by rwjblue** [comment]
Can you:

* add the following snippet to the top of each helper file:

```js
/**
@module ember
@submodule ember-testing
*/
```

* Remove the existing documentation in https://github.com/emberjs/ember.js/blob/master/packages/ember-testing/lib/helpers.js that this is replacing.

Thank you!

**Comment by witty123** [comment]
Done and done !

**Comment by rwjblue** [comment]
Looks great! Would you mind squashing the commits?

**Comment by witty123** [comment]
Squashed all the commits onto https://github.com/emberjs/ember.js/pull/13876/commits/1f548f7029c24432d058f26d1167950e76234fa5 
Didn't find any merge conflicts in helpers.js.

**Comment by witty123** [comment]
My bad. I think it's done now. Pardon me, I am new to git and github.

**Comment by rwjblue** [comment]
No worries! It looks like you did get this squashed, but it now includes many more changes than the ones you were working on. Can you take a stab at cleaning it up?

**Comment by witty123** [comment]
Hey, If it would not take much of your time could you explain me how I could have changed all those files. As I said, I am new to git and I am having a hard time trying to understand how those files got affected.

**Comment by mixonic** [comment]
Closed in https://github.com/emberjs/ember.js/pull/14107 - Thanks!

**Pull request closed without merging**